integrations/lambda/README.md

```diff
@@ -0,0 +1,108 @@
+- `requirements.txt`: A file containing required packages for the Jupyter notebook, for quick environment setup.
+
+The following diagram depicts the deployed Lambda function receiving generated data and ingesting the data to Timestream for LiveAnalytics that then is queried and displayed in [Amazon Managed Grafana](https://aws.amazon.com/grafana/).
+
+<img src="./img/lambda_ingestion_overview.png" width=575px/>
+
+## Prerequisites
+
+1. [Configure AWS credentials for use with boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html).
+2. [Install Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
+3. Initialize a Conda environment named `sample_app_env` with the required packages by running:
+    ```shell
+    conda env create -f environment.yml
+    ```
+4. Activate the environment by running:
+    ```shell
+    conda activate sample_app_env
+    ```
+5. Install an application to run the `demo.ipynb` file. We recommend [Visual Studio Code](https://code.visualstudio.com/) with the [Jupyter extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter).
+
+## Using the Jupyter Notebook Locally
+
+These steps assume you have followed the prerequisites and are using Visual Studio Code with the Jupyter extension.
+
+To run the notebook locally:
+
+1. After configuring the Conda environment, open `demo.ipynb` in Visual Studio Code.
+2. Click the search bar and from the dropdown menu select **Show and Run Commands**.
+3. Search for "Python: Select Interpreter".
+4. Select the Python "sample_app_env" environment from the dropdown menu.
+5. Scroll through the steps of the Jupyter notebook. Adjust values in the "Define Timestream for LiveAnalytics Settings" and "Generate Data" sections to your desire. Default values have been set for all steps.
+6. Once the kernel is running, press the **Run All** button.
+7. When all cells in the notebook have finished executing, records will have been ingested to the `sample_app_table` table in the `sample_app_database` database in Timestream for LiveAnalytics.
+
+## Using the Jupyter Notebook in Amazon SageMaker
+
+### IAM Configuration
+
+When deployed in Amazon SageMaker, the instance hosting the Jupyter notebook must use an IAM role with the following permissions:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "lambda:GetFunction",
+                "lambda:GetFunctionUrlConfig",
+                "lambda:InvokeFunctionUrl",
+                "lambda:UpdateFunctionUrlConfig",
+                "lambda:CreateFunction",
+                "lambda:CreateFunctionUrlConfig",
+                "lambda:AddPermission"
+            ],
+            "Resource": "arn:aws:lambda:<region>:<account ID>:function:TimestreamLambdaFunction"
```

Comment:
Good to be specific that the reader needs to replace account id and region with appropriate values.

Reply:
Added.